src/components/EmptyState/index.js

```diff
@@ -38,6 +39,9 @@ export class EmptyState extends React.Component {
           directionType={Stack.DIRECTION_TYPE.VERTICAL}
           gapType={Stack.GAP_TYPE.NONE}
         >
+          <StackItem>
```

Comment:
We should conditionally render this `<StackItem>` and it's contents depending on whether or not the user has supplied a `featuredImage`.